SONIC-DEV/sonic-buildimage
# [platform-device/celestica]: add seastone2 platform and device

**- What I did**
- Port device and platform code of seastone2 master branch to 201811 branch for azure.

**- How I did it**

- Add and modify code to support seastone2 device.
- Add and modify code to support seastone2 platform.

**- How to verify it**

- Check if the changes pass compilation.
- Run platform api test on testbed.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
